pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -50,7 +50,7 @@ func (s *Service) ReconcileLoadBalancer(openStackCluster *infrav1.OpenStackClust
 		return err
 	}
 	if lb == nil {
-		s.logger.Info("Creating load balancer", "name", loadBalancerName)
+		s.logger.Info("Creating load balancer", "name", loadBalancerName, "vipsubnetid", openStackCluster.Status.Network.Subnet.ID)
```

Comment:
I'm not sure if we should put all this data in key/value pairs. I think this makes mostly sense for fields where you can filter for later on through tools like Kibana/Loki. So I would only add the loadbalancer id (with key "id") as key value pair and the other fields into the message itself (although I would add the lb id to all those log lines so it's easy to filter on logs for this specific lb later on)

Reply:
ok, works for me , I only find it's really hard to find corresponding logs ..

Reply:
this one is for VM creation .. so the param is helpful for me to know what's given as param

Reply:
the others I only add the ID per suggestion , please let me know your opinoin

Reply:
I'm not against adding this info. I just wouldn't add it as key value pair :) I make an example:

```suggestion
		s.logger.Info(fmt.Sprintf("Creating load balancer in subnet: %q", openStackCluster.Status.Network.Subnet.ID), "name", loadBalancerName)
```

Reply:
Please see my other suggestions below. Something like this would make it possible to filter on the lb id to get all logs realted to as specific lb. If logs related to a specific listener should be found that can be still done by matching on the message.

I just think we shouldn' introduce arbitrary "dimensions" to our logs. I think key/value pairs in logs can be seen simlar to labels in metrics.